package.json

```diff
@@ -10,26 +10,30 @@
   },
   "scripts": {
     "start": "start-storybook -p 9001 -c .storybook",
-    "dev": "yarn start",
     "test": "cross-env CI=true ts-node ./scripts/test packages",
     "test:pkg": "cross-env CI=true ts-node ./scripts/test --pkg",
     "test:watch": "ts-node ./scripts/test packages",
     "test:coverage": "yarn test --coverage",
-    "build": "node -r ts-node/register --max-old-space-size=3049 scripts/build-all",
-    "build:pkg": "ts-node --transpile-only scripts/build-package",
+    "build": "yarn build:packages",
+    "build:packages": "preconstruct build",
     "ver": "lerna version --no-push --exact",
     "changes": "dotenv lerna-changelog",
     "clean": "git clean -e '!/.env' -e '!/website-deploy-key' -e '!/website-deploy-key.pub' -fdX .",
     "lint": "eslint .",
+    "fix": "preconstruct fix",
+    "dev": "yarn start",
+    "postinstall": "preconstruct dev",
```

Comment:
this pkg provides a nice & simple check so that your shared deps are always declared with the same dep range, highly recommend: https://github.com/Thinkmill/manypkg

Reply:
I mean - you could use it as part of postinstall script, like here: https://github.com/davidkpiano/xstate/blob/f36c5f84028e389bfb62e469f47253b7bce34d4f/package.json#L21